src/com/lilithsthrone/game/character/persona/Name.java

```diff
@@ -724,6 +796,10 @@ private static NameTriplet getDemonName() {
 	}
 	
 	public static NameTriplet getRandomProstituteTriplet() {
-		return Util.randomItemFrom(prostitute);
+		// occasionally throw some "regular" names in there - 25% of the time
+		if(Util.random.nextInt(3)<3)
```

Comment:
Also, I prefer all `if` `else` code blocks to be wrapped in parentheses, as the code can end up looking a little messy without them. ^^